devtest/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PARENT_IMAGE=openmicroscopy/omero-web-standalone:latest
+FROM ${PARENT_IMAGE}
+MAINTAINER ome-devel@lists.openmicroscopy.org.uk
+
+USER root
+
+RUN yum install -y -q \
+    java-11-openjdk \
+    patch \
+    redhat-lsb-core \
+    rsync \
+    tar \
+    unzip \
+    wget \
+    zip
+
+# Shouldn't be necessary any more since OMERO.web/bin/omero is a Python script but included anyway from previous image used in https://github.com/ome/omero-test-infra
```

Comment:
What would take to strip it from test-infra?

Reply:
I don't know, though I suspect it might already work. My intention in this PR was to minimise changes that might impact omero-test-infra, but if you're happy with larger changes I can try removing it.